### PR TITLE
ncurses 6.3

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Bo98** [comment]
Reading the code, `--with-pkg-config-libdir` should work fine but the directory must exist before `./configure` is called.

**Review by SMillerDev — DISMISSED** [review]

**Comment by carlocab** [comment]
Most of the Linux failures are Java-related, and being handled at

There are some that are service-related. CC @SMillerDev
```
==> Pouring burp--2.2.18_1.x86_64_linux.bottle.tar.gz
Error: Failed to install service files
Error: undefined method `working_directory' for #<Homebrew::Service:0x0000000003329238>
Did you mean?  working_dir
```

**Comment by carlocab** [comment]
There's also
```
==> Pouring jruby--9.3.0.0.x86_64_linux.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /home/linuxbrew/.linuxbrew
Could not symlink bin/bundle
Target /home/linuxbrew/.linuxbrew/bin/bundle
is a symlink belonging to ruby. You can unlink it:
  brew unlink ruby
```

File: Formula/ncurses.rb [hunk 1]
```diff
@@ -47,17 +56,11 @@ def install
...
-        lib.install_symlink "lib#{name}w.#{major}.dylib" => "lib#{name}.dylib"
-        lib.install_symlink "lib#{name}w.#{major}.dylib" => "lib#{name}.#{major}.dylib"
-      end
-      on_linux do
-        lib.install_symlink "lib#{name}w.so.#{major}" => "lib#{name}.so"
-        lib.install_symlink "lib#{name}w.so.#{major}" => "lib#{name}.so.#{major}"
-      end
+    major = version.major.to_s
+
+    %w[form menu ncurses panel curses++].each do |name|
```
Comment (carlocab) [inline Formula/ncurses.rb:61]:
```suggestion
    %w[form menu ncurses panel ncurses++].each do |name|
```
Oops.

**Comment by carlocab** [comment]
Rerunning this but skipping dependents because I made a typo (see suggestion above). Linux CI log from the dependents run here: https://github.com/Homebrew/homebrew-core/runs/4148563254?check_suite_focus=true

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**